lib/src/buttons/primary_button.dart

```diff
@@ -103,6 +104,7 @@ class _PrimaryButtonState extends State<MUIPrimaryButton> {
         decoration: BoxDecoration(
           borderRadius: BorderRadius.circular(widget.borderRadius),
           color: widget.bgColor,
+          boxShadow: const [BoxShadow(blurRadius: 5)],
```

Comment:
```dart
BoxShadow(
    color: Color.fromRGBO(72, 76, 82, 0.16);,
    offset: const Offset(0, 12),
    blurRadius: 16.0,
)
```

Please have optional parameter in constructor for shadow, When It's not passed from constructor then use above snippet of `BoxShadow` as default shadow otherwise passed shadow from constructor.